bety_brapi/controllers/germplasm_controller.py

```diff
@@ -193,3 +193,15 @@ def studies_study_db_id_germplasm_get(studyDbId, pageSize=None, page=None):  # n
 #    return GermplasmController_impl.studies_study_db_id_germplasm_get(studyDbId, pageSize=None, page=None)
     return "Not implemented"
 
+
+def treatments_by_experiment_get(experimentId):
+
```

Comment:
remove empty line between function and first code line

Reply:
removed.